caffe2/CMakeLists.txt

```diff
@@ -179,6 +199,35 @@ if(FALSE)
     message(STATUS "  " ${tmp})
   endforeach()
 
+  if(BUILD_SPLIT_CUDA)
+    message(STATUS "GPU sources: (for torch_cuda_cpp)")
+    foreach(tmp ${Caffe2_GPU_SRCS_CPP})
+      message(STATUS "  " ${tmp})
+    endforeach()
+
+    message(STATUS "GPU sources: (for torch_cuda_cu)")
+    foreach(tmp ${Caffe2_GPU_SRCS_CU})
+      message(STATUS "  " ${tmp})
+    endforeach()
+  endif()
+
+  message(STATUS "GPU sources (w/ sort by key): ")
```

Comment:
nit: did you mean to cover this print by `if(not BUILD_SPLIT_CUDA)` or are you actually going to print all 3
`Caffe2_GPU_SRCS_W_SORT_BY_KEY`
`Caffe2_GPU_SRCS_W_SORT_BY_KEY_CU`
`Caffe2_GPU_SRCS_W_SORT_BY_KEY_CPP`
when BUILD_SPLIT_CUDA is turned on?

btw have you tested this section locally by enabling the debug msg?

Reply:
LOL I actually used this to debug so I have tested it--I wanted to print all three to make sure that Caffe2_GPU_SRCS_W_SORT_BY_KEY_CU union Caffe2_GPU_SRCS_W_SORT_BY_KEY_CPP is Caffe2_GPU_SRCS_W_SORT_BY_KEY.